doc/fastfetch.1.in

```diff
@@ -0,0 +1,146 @@
+.TH FASTFETCH 1 "@FASTFETCH_BUILD_DATE@" "@CMAKE_PROJECT_NAME@ @CMAKE_PROJECT_VERSION@"
+
+.SH NAME
+fastfetch - a neofetch-like tool for fetching system information and
+displaying them in a pretty way
+
+.SH SYNOPSIS
+
+.B fastfetch
+.I [options...]
+
+.SH DESCRIPTION
+
+Fastfetch is a neofetch-like tool for fetching system information and
+displaying them in a pretty way. It is written mainly in C, with performance
+and customizability in mind.
+Currently, Linux, Android, FreeBSD, MacOS and Windows 7+ are supported.
+
+.SH "EXIT STATUS"
+
+On successfull execution, fastfetch returns zero. If any error happened,
+the exit code will be non-zero.
+
+.SH OPTIONS
+
+.SS "Informative Options"
+.TP
+
+.B \-\-help \fI [command]
+Show help output, displaying all available options or help for given command
+.TP
+
+.B \-\-version
+Show version of fastfetch
+.TP
+
+.B \-\-list\-config\-paths
+List search paths of config files
+.TP
+
+.B \-\-list\-data\-paths
+List search paths of presets and logos
+.TP
+
+.B \-\-list\-logos
+List available logos, they can be loaded with \fI \-\-logo
+.TP
+
+.B \-\-list-modules
+List available modules
+.TP
+
+.B \-\-list-presets
+List available presets, they can be loaded with \fI \-\-load\-config
+.TP
+
+.B \-\-list-features
+List the supported features fastfetch was compiled with
+.TP
+
+.B \-\-print-logos
+Print available logos
+.TP
+
+.B \-\-print\-default\-structure
```

Comment:
It should be `--print-structure`

Reply:
fixed in 5445ef1581ab2cc8b5366fa49696225f555e3a61

Also fixed help.txt in 1cd8235c2927ab0979197b14d3fecdf76fb9dec1